src/main/java/teammates/logic/core/FeedbackQuestionsLogic.java

```diff
@@ -566,7 +566,7 @@ public void updateFeedbackQuestionNumber(FeedbackQuestionAttributes newQuestion)
         }
         
         adjustQuestionNumbers(oldQuestionNumber, newQuestionNumber, questions);
-        updateFeedbackQuestion(newQuestion);
+        updateFeedbackQuestionWithResponseRateCheck(newQuestion);
```

Comment:
I have the same idea too, the old `updateFeedbackQuestion` should be made private. The comment should also mention that the response rate is not updated.